### PR TITLE
OrgPicker: Fixes so organisations load

### LINKED ISSUE #33981: Can not add users to an organization from Server Admin page
**What happened**:

As a Server Admin I am creating users and multiple organizations and I want to add users to different organizations. When I try to do that in the Server Admin UI, I can't select previously added orgs in the UI, thus adding users to orgs is not possible.

**What you expected to happen**:
Server Admin should see all available orgs in the Server Admin UI and able to select to add users.

**How to reproduce it (as minimally and precisely as possible)**:
1. As a Server Admin, create multiple organizations
2. Create a new user from the Server Admin page
3. After user is created, go to the user profile page from the Server Admin users list view
4. Under Organizations, click "Add user to organization" button
5. From the opened modal, select the dropdown to list Organizations
6. The list is empty, although we have created several orgs in the first step

Here is a screenshot
Screenshot 2021-05-12 at 11 40 33

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
I made a mistake when fixing strict null errors that prevented the OrgPicker from loading because it's now initialized to an empty array.

**Which issue(s) this PR fixes**:
Fixes #33981

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/core/components/Select/OrgPicker.tsx [hunk 1]
```diff
@@ -36,7 +36,7 @@ export class OrgPicker extends PureComponent<Props, State> {
   }
 
   getOrgOptions = async (query: string): Promise<Array<SelectableValue<number>>> => {
-    if (!this.orgs) {
+    if (!this.orgs || !this.orgs.length) {
```
Comment (Clarity-89) [inline public/app/core/components/Select/OrgPicker.tsx:39]:
```suggestion
    if (!this.orgs?.length) {
```
This should work as well I think.

**Review by Clarity-89 — APPROVED** [review]

File: public/app/core/components/Select/OrgPicker.tsx (hunk 1 above)
Comment (hugohaggmark) [reply public/app/core/components/Select/OrgPicker.tsx:39]: Beautiful, 🙇

**Review by torkelo — APPROVED** [review]
Can you make the Modal use Modal.ButtonRow and the cancel button use secondary & fill="outline" ?

**Review by dprokop — APPROVED** [review]

**Comment by hugohaggmark** [comment]
> Can you make the Modal use Modal.ButtonRow and the cancel button use secondary & fill="outline" ?

I can and will!

**Comment by torkelo** [comment]
and move cancel to be first (left) of primary button

**Pull request merged**